cmd/virt-launcher/virt-launcher.go

```diff
@@ -499,7 +502,7 @@ func main() {
 
 	domain := waitForDomainUUID(*qemuTimeout, events, signalStopChan, domainManager)
 	if domain != nil {
-		mon := virtlauncher.NewProcessMonitor(domain.Spec.UUID,
+		mon := virtlauncher.NewProcessMonitor("uuid="+domain.Spec.UUID,
```

Comment:
nice trick :)

Reply:
@jean-edouard IIUC this should resolve the issue #7067, I think this change should go better in a separate commit with description about how does this change resolve it.

Reply:
This is really a band-aid, I wouldn't say it's a proper fix for that issue...

Reply:
@jean-edouard IMHO I would mark this is some way that its actually a short-term solution, and describe the actual problem its trying to solve, but its up to you, I won't insist

Reply:
Added a `FIXME` comment